external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -505,7 +505,7 @@ abstract class KafkaSinkBatchSuiteBase extends KafkaSinkSuiteBase {
     testUtils.createTopic(topic)
     val df = Seq[(String, String)](null.asInstanceOf[String] -> "1").toDF("topic", "value")
 
-    Seq(SaveMode.Ignore, SaveMode.Overwrite).foreach { mode =>
+    Seq(SaveMode.Overwrite).foreach { mode =>
```

Comment:
Why did this Kafka test change? The source doesn't implement the new interface, so its behavior should be the same?